src/Vehicle/Vehicle.cc

```diff
@@ -3660,6 +3661,23 @@ void Vehicle::_updateDistanceToGCS()
     }
 }
 
+void Vehicle::_updateHomepoint()
+{
+    if(_toolbox->settingsManager()->flyViewSettings()->updateHomePosition()->rawValue().toBool()
```

Comment:
Maybe saving these values in variables such as updateHomePosition and supportsSetHome helps with readability.

Reply:
see commit https://github.com/mavlink/qgroundcontrol/pull/10085/commits/a743b4ca277e862b89333743b50e40e8647a9444